packages/tsx-editor/src/transpiler/__snapshots__/exampleTransform.test.ts.snap

```diff
@@ -1,8 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`example transform handles examples with class components 1`] = `
-"const {SpinButton, Fabric } = window.Fabric;
+"const {
```

Comment:
Nitpicky thing but it would be nice if these imports were either all on one line, or each on their own line.

Reply:
I think I'd prefer each on their own line since this is the version of the output that will be used if they export to codepen, and having each import on its own line makes it easier to read/edit.